### DISCUSSION TITLE
How to keep style that is copied and pasted?

### DISCUSSION THREAD

**Discussion by deliqs** [singerdmx/flutter-quill#1753] [General]
One of my users reported that the styling (like white lines) he copies and pastes in the Quill editor doesn't retain, which was possible previously. Does anyone know how I can enable this again?

He copies the text from Evernote
![SCR-20240224-kjyx](https://github.com/singerdmx/flutter-quill/assets/88924253/0bcccbcf-94a5-425b-aa24-6a84a9df75df)

into our app. And it loses the white lines.
![Screenshot_20240223_234021](https://github.com/singerdmx/flutter-quill/assets/88924253/8472d03f-9e29-4d8e-aecd-5a923e2ceccd)

**Comment by CatHood0** [comment]
Is this still happening?
